examples/terrain_tessellated/shader/terrain.glslg

```diff
@@ -0,0 +1,60 @@
+#version 150 core
+
+layout(triangles) in;
+layout(triangle_strip, max_vertices=3) out;
+
+layout (std140)
+uniform Locals {
+	mat4 u_Model;
+	mat4 u_View;
+	mat4 u_Proj;
+};
+
+in block {
+    vec3 pos;
+	vec3 color;
+} v_In[];
+
+out vec3 v_Color;
+
+out gl_PerVertex {
+	vec4 gl_Position;
+	float gl_PointSize;
+	float gl_ClipDistance[];
+};
+
+struct V {
+	vec3 pos;
+	vec3 color;
+};
+
+V read_vertex(int i) {
+	V v;
+	v.pos = v_In[i].pos;
+	v.color = v_In[i].color;
+	return v;
+}
+
+void emit_vertex(V v) {
+	v_Color =  v.color;
+	gl_Position = u_Proj * u_View * u_Model * vec4(v.pos, 1.0);
+    gl_ClipDistance[0] = 1.0;
```

Comment:
nit: indentation

Reply:
Argh, ffs. I don't have autoformat for glsl. Will fix manually.

Reply:
Fixed indentation.